### DISCUSSION TITLE
Advance Search (Find product easy and fast)

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5271] [Working on Features]
Integrate Fluxstore with  [https://tr.wordpress.org/plugins/relevanssi/](https://tr.wordpress.org/plugins/relevanssi/)

ıts help us on the easy and correct search 

**With this plugin, our customers will be able to reach the product they are looking for even if they make a typo, and this will positively affect our sales.**

this plugın have 

- Search results are sorted in the order of relevance, not by date.
- Fuzzy matching: match partial words, if complete words don’t match.
- Find documents matching either just one search term (OR query) or require all words to appear (AND query).
- Search for phrases with quotes, for example, “search phrase”.
- Create custom excerpts that show where the hit was made, with the search terms highlighted.
- Highlight search terms in the documents when the user clicks through search results.
- Search comments, tags, categories, and custom fields.
- Multisite friendly.
- bbPress support.
- Gutenberg friendly.

**Comment by minhcasi** [comment]
Hi please refer to the Ajax Search Pro feature to request via ajax to archive this plugin - thank you!

**Reply by osman-kocak** [reply]
YITH WooCommerce Featured Video not work with Ajax Search Pro

***WooCommerce Featured Video = This plugin has been closed as of January 6, 2022 and is not available for download.

**Comment by usamaahmed5656** [comment]
if you are not on shared hosting than all of the above is acheiveable by elastic press plugin which needs elastic search to be installed on your server 

elastic press is already compatible with all plugins and also working fine with all fluxstore
